core/src/main/java/nl/nn/adapterframework/configuration/ConfigurationUtils.java

```diff
@@ -99,13 +99,11 @@ public static String getActivatedConfiguration(Configuration configuration, Stri
 		return getTweakedConfiguration(configuration, originalConfig, ACTIVE_XSLT, null);
 	}
 
-	public static String getUglifiedConfiguration(Configuration configuration, String originalConfig) throws ConfigurationException {
-		return getTweakedConfiguration(configuration, originalConfig, UGLIFY_XSLT, null);
+	public static String getCanonicalizedConfiguration(Configuration configuration, String originalConfig) throws ConfigurationException {
+		return getTweakedConfiguration(configuration, originalConfig, CANONINICALIZE_XSLT, null);
```

Comment:
Zullen we getTweakedConfiguration ook hernoemen? Hij Tweaked niets?

Reply:
De getStubbedConfiguration, getActivatedConfiguration en getUglifiedConfiguration zijn min of meer voorbeelden van tweaks. De term is ontstaan toen we voor HEC migratie custom tweaks op de configs toe pasten. Het idee was/is om er iets generieks van te maken en ontwikkelaars in staat te stellen 1 of meerdere tweaks op hun configuratie te doen. De getStubbedConfiguration, getActivatedConfiguration en getUglifiedConfiguration zijn dan standaard/ingebouwde tweaks maar een Frank! ontwikkelaar zou zijn eigen tweak ook nog toe kunnen voegen.

Reply:
Mijn idee was ook om bij de show configuration alle verschillende transformaties te laten zien, dus niet alleen Original Configuration en Loaded Configuration maar een tabje voor het restultaat van iedere "ConfigTweak".